src/chat.rs

```diff
@@ -3022,15 +3022,7 @@ pub async fn set_chat_profile_image(
         msg.param.remove(Param::Arg);
         msg.text = Some(stock_str::msg_grp_img_deleted(context, ContactId::SELF).await);
     } else {
-        let mut image_blob = match BlobObject::from_path(context, Path::new(new_image.as_ref())) {
-            Ok(blob) => Ok(blob),
-            Err(err) => match err {
-                BlobError::WrongBlobdir { .. } => {
-                    BlobObject::create_and_copy(context, Path::new(new_image.as_ref())).await
```

Comment:
This should be redone without using error as a return value.

Reply:
to not end up with a nonfunctional `set_chat_profile_image()` this should be fixed before this pr is merged.

having a quick look, maybe the whole `match` can be replaced by `new_from_path()` - this is how selfavatar is set.